packages/UsageAnalysis/files/Test Track/General/Api samples.md

```diff
@@ -0,0 +1,11 @@
+Since the API examples' automated tests are currently disabled, we need to verify them manually.
+1. Go Apps > TestManager. Press 'Run' button.
+2. Run "Api Samples".
+3. Expected result: all tests are passed successfully without errors in console.
```

Comment:
I don't think we can expect all Api Samples ran in the test manager to just work. Does it work this way?

Reply:
Changed to more appropriate